config/manifests/bases/smart-gateway-operator.clusterserviceversion.yaml

```diff
@@ -69,8 +69,8 @@ metadata:
     createdAt: <<CREATED_DATE>>
     description: Operator for managing the Smart Gateway Custom Resources, resulting
       in deployments of the Smart Gateway.
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.10"}]'
     olm.skipRange: =><<BUNDLE_OLM_SKIP_RANGE_LOWER_BOUND>> <<<OPERATOR_BUNDLE_VERSION>>
-    "olm.properties": '[{"type": "olm.maxOpenShiftVersion", "value": "4.8"}]'
     operators.operatorframework.io/builder: operator-sdk-v0.19.4
```

Comment:
Expected to see this change to v1.16.0

Reply:
Yes please, we are using these annotations in some of our data to see the traction of operator-sdk

Reply:
Oops! totally missed this. The CSV wasn't re-generated really. I will update this now!

Reply:
(I'm going to generate a stubbed CSV and make sure I copy over any changes or additional annotations.